cos-fleetshard-operator-debezium/pom.xml

```diff
@@ -119,6 +119,13 @@
             <plugin>
                 <groupId>io.quarkus</groupId>
                 <artifactId>quarkus-maven-plugin</artifactId>
+                <dependencies>
+                    <dependency>
+                        <groupId>org.bf2</groupId>
+                        <artifactId>cos-fleetshard-api-deployment</artifactId>
+                        <version>${cos-fleetshard-api.version}</version>
```

Comment:
can this be ${project,version} ?

Reply:
I don't know. This is a property that we already have and use everywhere. Maybe it's a valid use case to progress with the api package code without using the newer version at the same time? This is just how it is specified in the `cos-fleetshard` dependencyManagement section.

Reply:
The `cos-fleetshard-api` is a module of the project and defines the CRD of the operator
I don't see a good use case to keep it on a different version.

Reply:
Done already